src/Reconstruction/RecoParameters.jl

```diff
@@ -125,7 +125,7 @@ function setupIterativeReco(acqData::AcquisitionData{T}, recoParams::Dict) where
   # noise data acquisition [samples, coils]
   noiseData = get(recoParams, :noiseData, Complex{T}[])
   if isempty(noiseData)
-    L_inv = noiseData
+    L_inv = []
```

Comment:
I think this change is good, explicitly initializing it as empty. Could also potentially initialize it as Nothing?